toolbox/oidc_connect.go

```diff
@@ -34,15 +32,20 @@ type oidcToken struct {
 type oidcTokenProvider struct {
 	timeout           time.Duration
 	perRealmTokenInfo map[string]*oidcTokenInfo
-	reqBody           string
-	defaultKey        string
-	logger            Logger
+	//reqBody           string
+	username   string
+	password   string
+	defaultKey string
+	logger     Logger
 }
 
 type oidcTokenInfo struct {
-	url        string
-	oidcToken  oidcToken
-	validUntil int64
+	//url         string
+	forwarded    string
+	oauth2Config *oauth2.Config
+	tokenSource  oauth2.TokenSource
+	//oidcToken  oidcToken
+	//validUntil int64
```

Comment:
Leftovers comment ?

Reply:
Left only in case you asked me to get back to previous "home made" code.